bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java

```diff
@@ -30,6 +30,8 @@
 public abstract class AbstractTest {
   protected DataGenerationHelper dataHelper = new DataGenerationHelper();
 
+  protected Logger logger = new Logger(this.getClass());
```

Comment:
When looking at a log, I feel that it's important to be able to be able to quickly tell what class produced a log message.